.github/CODEOWNERS

```diff
@@ -1452,6 +1456,10 @@ packages/react @elastic/appex-sharedux
 /packages/core/saved-objects/docs/openapi     @elastic/platform-docs
 /plugins/data_views/docs/openapi              @elastic/platform-docs
 
+# Plugin manifests
+/src/plugins/**/kibana.jsonc @elastic/kibana-core
+/x-pack/plugins/**/kibana.jsonc @elastic/kibana-core
```

Comment:
Adding the team as codeowners of the (production) plugin manifests. Need to be at the bottom of the file to make it it takes priority over per-plugin rules.